README.md

```diff
@@ -88,14 +89,26 @@ You now have four ways to install APISIX:
 
 *NOTE*: APISIX currently only supports the v2 protocol storage to etcd, but the latest version of etcd (starting with 3.4) has turned off the v2 protocol by default. You need to add `--enable-v2=true` to the startup parameter to enable the v2 protocol. The development of the v3 protocol supporting etcd has begun and will soon be available.
 
+### Install runtime dependency
+
+The runtime of APISIX supports both OpenResty and Tengine. You can choose any one you would like. The current test case has test coverage for both platforms.
+
+1.  openresty & etcd
+
+    ```shell
+    sudo yum install yum-utils
+    sudo yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
+    sudo yum install -y openresty etcd
+    sudo service etcd start
+    ```
+
```

Comment:
this part is ONLY for centos, should in the `Install from RPM for CentOS 7` section.